hack/test-end-to-end.sh

```diff
@@ -205,6 +205,10 @@ openshift admin policy add-role-to-user view e2e-user --namespace=default
 # pre-load some image streams and templates
 oc create -f examples/image-streams/image-streams-centos7.json --namespace=openshift
 oc create -f examples/sample-app/application-template-stibuild.json --namespace=openshift
+oc create -f examples/jenkins/application-template.json --namespace=openshift
```

Comment:
I wanted to have more template options instead of just ruby-helloworld-sample, trying to be closer to a real-world scenario.